src/patterns-hmrc-govuk-team/Insolvency practitioner bulletins/index.njk

```diff
@@ -0,0 +1,61 @@
+---
+title: Insolvency practitioner bulletins
+menuText: Insolvency practitioner bulletins
+layout: twoColumnPage.njk
+---
+
+<h2 class="govuk-heading-l">When to use</h2>
+
+<p class="govuk-body">Use this pattern when creating or amending insolvency practitioner bulletins.</p>
+
+<h2 class="govuk-heading-l">How to design using this pattern</h2>
+
+<h3 class="govuk-heading-m">Document type</h3>
+
+<p class="govuk-body">Publication: guidance (with an HTML attachment)</p>
+
+<h3 class="govuk-heading-m">Publication</h3>
+
+<h4 class="govuk-heading-s">Title</h4>
+
+<p class="govuk-body">The title should be: Insolvency practitioner bulletin [issue number] [(year)]: [insert title]</p>
+<p class="govuk-body">For example, Insolvency practitioner bulletin 9 (2024): changes to the insolvency customer services mailbox</p>
+
+<h4 class="govuk-heading-s">Summary</h4>
+
+<p class="govuk-body">Begin with:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>This bulletin explains ... [then read the document and summarise the content].</li>
+  <li>This bulletin tells you about … [then read the document and summarise the content].</li>
+  <li>This bulletin gives an update ... [then read the document and summarise the content].</li>
+</ul>
+<p class="govuk-body">For example, ‘This bulletin tells you about the new rules when sending emails to the mailbox, and how it will support you with repayments or refund requests.’</p>
+
+<h4 class="govuk-heading-s">Details</h4>
+
+<p class="govuk-body">Begin with:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>This bulletin explains [insert detail].</p>
+  
+  <p class="govuk-body">or</p>
+  <li>This bulletin explains:<br>[detail]<br>[detail]</li>
+</ul>
```

Comment:
Only `<li>`, `<script>` and `<template>` elements are allowed in `<ul>`s (https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/ul#technical_summary) could you change to something like this please:
```suggestion
<ul class="govuk-list govuk-list--bullet">
  <li>This bulletin explains [insert detail].</li>
</ul>
  
<p class="govuk-body">or</p>

<ul class="govuk-list govuk-list--bullet">
  <li>This bulletin explains:<br>[detail]<br>[detail]</li>
</ul>
```